nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -625,7 +629,7 @@ private void setAuthenticator(OkHttpClient.Builder okHttpClientBuilder, ProcessC
             final DigestAuthenticator digestAuthenticator = new DigestAuthenticator(credentials);
 
             if(!proxyUsername.isEmpty()) {
-                final String proxyPassword = context.getProperty(PROP_PROXY_PASSWORD).getValue();
+                final String proxyPassword = context.getProperty(PROP_PROXY_PASSWORD).evaluateAttributeExpressions().getValue();
```

Comment:
This precludes "plain-text" passwords with valid EL inside them, but I think it's getting more standard to support these, and eventually we will have encrypted variables, so I'm just mentioning it for posterity ;)